packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -1878,4 +1878,103 @@ describe('@api-batch-3 RPC Server Acceptance Tests', function () {
       });
     });
   });
+
+  describe('Batch Request Test Suite', async function () {
+    // make sure env is set to true BATCH_REQUESTS_ENABLED
```

Comment:
Instead of comment you can just set it.
Maybe add a before and after.
See the `describe('DEV_MODE = true', ...` example

Reply:
Thanks for the heads up on this